WordPress/Classes/ViewRelated/Comments/CommentContentTableViewCell.swift

```diff
@@ -168,4 +265,7 @@ private extension String {
                                                         + "%1$d is a placeholder for the number of Likes.")
     static let pluralLikesFormat = NSLocalizedString("%1$d Likes", comment: "Plural button title to Like a comment. "
                                                 + "%1$d is a placeholder for the number of Likes.")
+
+    // pattern that detects empty HTML elements (including HTML comments within).
+    static let emptyElementRegexPattern = "<[a-z]+>(<!-- [a-z\\/:]+ -->)+<\\/[a-z]+>"
```

Comment:
I did find some parameters embedded in the HTML comments, e.g.
```html
<!-- wp:image {"id":10675,"sizeSlug":"large","linkDestination":"none"} -->
```
which escapes the regex pattern. Will fix this tomorrow!

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/17129/commits/2c0ae96da04e1f7de9dedbce7851f4be27ca7ccc.